Jenkinsfile

```diff
@@ -97,6 +97,7 @@ pipeline {
       steps {
         dir(env.REPO_DIR) {
           sh "npm run release"
+          sh "npm run sync-versions"
```

Comment:
I decoupled these commands to make it easier (do it in jenkins without a need for a PR like this) to bump the version when needed.